rest-e2e-tests/products-api.js

```diff
@@ -12,6 +12,11 @@ const DEVOPS_USER_TOKEN = 'devops-user-key'
 const Streamr = initStreamrApi(URL, LOGGING_ENABLED)
```

Comment:
Rename tests to *.test.js

Reply:
If it's okay with you, I'd like to do the renaming after a couple of branches have been landed on marketplace-api. So then I can do it in a single swoop commit.

Reply:
I've written a note for myself for this.

Reply:
👍